OpenLIFUHome/OpenLIFUHome.py

```diff
@@ -98,9 +96,15 @@ def setup(self) -> None:
         # Make sure parameter node is initialized (needed for module reload)
         self.initializeParameterNode()
 
+        # Set the initial guided mode status
+        self.logic.getParameterNode().guided_mode = False
+
```

Comment:
Instead of initializing in setup, I think you should do
```py
@parameterNodeWrapper
class OpenLIFUHomeParameterNode:
    guided_mode : bool  = False
```
up above. (see [parameter node defaults docs](https://slicer.readthedocs.io/en/latest/developer_guide/parameter_nodes/defaults.html) -- if i understand correctly this will make it initialize to False)

One advantage of doing it that way is that whether or not we are in guided mode can persist over scene save and load. If guided mode is set to False in the module setup then it would be forced into false if you ever loaded a scene that was saved with guided mode enabled (I think)